xchange-coinsph/pom.xml

```diff
@@ -44,6 +44,12 @@
             <version>${version.lombok}</version>
             <scope>compile</scope>
         </dependency>
+
```

Comment:
please sort pom by building it with maven

Reply:
@bigscoop apologies I don't quite understand what you're asking

Reply:
cool done